src/App.tsx

```diff
@@ -1,13 +1,38 @@
+import { useEffect, useState } from "react";
 import styles from "./App.module.css";
+import { faker } from "@faker-js/faker";
+import FlightTable from "./components/FlightTable/FlightTable";
 
 export interface Flight {
   // ... fields here
+  flightNumber: string;
+  origin: string;
+  destination: string,
+  departureTime: string,
+  arrivalTime: string,
+  flightDuration: number
 }
 
 const App = () => {
   // state for flights should go here
+  const [flights, setFlights] = useState<Flight[]>([]);
+  useEffect(() => {
+    const newFlights = flights.sort((a: Flight, b: Flight) => ( a.flightDuration - b.flightDuration ));
+    console.log(newFlights);
+    setFlights(newFlights);
+  }, flights)
```

Comment:
This is great! Great job with the `.sort(...)` function, we'll use that a lot this year